features/FEATURE_BLE/targets/TARGET_NORDIC/TARGET_MCU_NRF51822/source/nRF5xGattServer.cpp

```diff
@@ -65,6 +65,7 @@ ble_error_t nRF5xGattServer::addService(GattService &service)
             return BLE_ERROR_NO_MEM;
         }
         GattCharacteristic *p_char = service.getCharacteristic(i);
+        GattAttribute *description_descriptor = NULL;
```

Comment:
should use the existing code practice of prefixing the pointer with p_

Reply:
Fixed